### ISSUE TITLE
Add  roms automatically after scanning folder(no metadata setup etc)

### ISSUE THREAD

**Issue by ValeriyKoloytanov** [mmatyas/pegasus-frontend#483]
First of all I want to say that I really appreciate your work.This project has come a long way, it is very impressive.Back to feature request. Please make this frontend more beginner friendly.Creating and metadata files can be hard at times

**Comment by mmatyas** [comment]
Yes, I agree; at the moment I'm working on making most features work correctly (eg. gamepad support, menu improvements, themes), then I do plan to create some kind of [setup screen](#120) and update the [documentations](https://github.com/mmatyas/pegasus-frontend/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation) with tutorials or videos. There are also tools like the [metadata editor](https://github.com/mmatyas/pegasus-metadata-editor/releases/tag/continuous) which I'd like to flesh out a bit. It just takes time :)
